dspace/config/registries/journal-types.xml

```diff
@@ -86,4 +86,14 @@
         <element>manuscriptNumberIgnorePattern</element>
     </dc-type>
 
+    <dc-type>
+        <schema>journal</schema>
+        <element>description</element>
+    </dc-type>
+
+    <dc-type>
+        <schema>journal</schema>
+        <element>memberName</element>
```

Comment:
sponsorName is already listed in the file, above at line 57. I added journal.memberName item so that the journal metadata panel at the top of the landing page could include both "Member:" and "Sponsorship:" fields.